code/game/objects/items/weapons/stunbaton.dm

```diff
@@ -7,15 +7,30 @@
 	slot_flags = SLOT_BELT
 	force = 10
 	throwforce = 7
-	w_class = WEIGHT_CLASS_NORMAL
 	origin_tech = "combat=2"
 	attack_verb = list("beaten")
 	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 50, "bio" = 0, "rad" = 0, "fire" = 80, "acid" = 80)
+	/// How many life ticks does the stun last for
 	var/stunforce = 7
-	var/status = 0
-	var/obj/item/stock_parts/cell/high/cell = null
+	/// Is the baton currently turned on
+	var/enabled = FALSE
```

Comment:
The fourteenth time I say this. I know.

How about you just change the variable name to `turned_on`? Relying on comments instead of proper naming leads to less readable code. This applies in many other places in this PR, but I won't spam comments.

Reply:
I definitely agree with this, the only reason it wasn't like that in the first place was because it just never crossed my mind.